### ISSUE TITLE
Debugging a program through OpenOCD

### ISSUE THREAD

**Issue by Chakri-myneni** [riscv-software-src/riscv-isa-sim#223]
I am trying to debug a program through openOCD following the steps from [https://github.com/riscv/riscv-isa-sim](url)

Files used for debugging are rot13.c, spike.cfg from the above mentioned link.
After compiling,
-> For **spike --rbb-port=9824 -m0x10000000:0x20000 rot13-64** this command, it is listening on port 9824.
![spike_run_1](https://user-images.githubusercontent.com/42030248/43576767-c130a120-9667-11e8-92f5-c99b113140ef.JPG)

-> For **openocd -d -f spike.cfg** this command the result is as follows
![openocd_2](https://user-images.githubusercontent.com/42030248/43576381-d432d8ac-9666-11e8-8ed5-8c4cd2148533.png)

Tool chain is latest.(i.e updated and built on 2nd Aug 2018)

My questions are 
1. what can be the reason for the error ?
2. what are the necessary changes to be made?
3. Do I need to enable something while building the toolchain? If so what should be enabled?

**Comment by timsifive** [comment]
See discussion at https://forums.sifive.com/t/debugging-a-program-through-openocd/1471

**Comment by Chakri-myneni** [comment]
@timsifive that question in sifive forums is also posted by me (Chakri).

**Comment by benjschro** [comment]
For what it is worth, I am running into this EXACT same problem.  For the whole toolchain and additional tools, I cloned them from the git repos a couple days ago.  I am running this all in ubuntu-18.04.1-live-server-amd64 in a virtual machine.

Edit - Nevermind.  I've fixed my problem.  It wasn't obvious to me but I compiled openocd without an important switch.  After I did so, everything works.

For OpenOCD
 ./configure --prefix=/opt/riscv --enable-remote-bitbang
